src/include/miopen/conv/data_invoke_params.hpp

```diff
@@ -38,15 +38,15 @@ struct DataInvokeParams : InvokeParams
     Data_t workSpace;
     std::size_t workSpaceSize;
     bool gfx90aFp16alt;
-    const void* alpha;
-    const void* beta;
+    ConstData_t alpha;
```

Comment:
`Data_t` and `ConstData_t` in our codebase refer to buffer types. We shouldn't use that for scalars. I made a comment in #2817 where I want wrapper classes for `Scalar` parameters that contain data type info as well.